Intial commits

katakampranavi opened the pull request:
![sunbird-framework-management-tool-snapshot]- https://drive.google.com/file/d/1kdX56m_iqZAEbhI9EYB3T2ozhVmP4QQ7/view?usp=sharing

katakampranavi committed:
Create test

katakampranavi committed:
intial changes

HarishGangula commented:
The footer will be bottom of the page

HarishGangula commented:
replace logo icon from. react to sunbird

HarishGangula commented on sunbird-framework-management-tool/src/App.js:
```diff
@@ -0,0 +1,40 @@
+        </Typography>
+      </Container>
+      <footer
+        style={{
```
No inline style allowed

katakampranavi committed:
Changes Made

changes made as suggested
![Sunbird-framework-image] https://drive.google.com/file/d/1TD_1x0MdLSQ9L3baRBrTy-ywBbB-t6YK/view?usp=sharing

katakampranavi commented:
@HarishGangula As suggested i made the changes please find below link for snapshot
https://drive.google.com/file/d/1TD_1x0MdLSQ9L3baRBrTy-ywBbB-t6YK/view?usp=sharing

HarishGangula commented on sunbird-framework-management-tool/public/index.html:
```diff
@@ -0,0 +1,43 @@
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Change this title to `sunbird framework management tool`

katakampranavi committed:
changes made as suggested

katakampranavi commented:
@HarishGangula title name changed from react to sunbird framework management tool
https://drive.google.com/file/d/1XFUe1CGMlOE9ZFUMYbQ4kppI45CXUwK1/view?usp=sharing
